.github/workflows/benchmarking.yml

```diff
@@ -0,0 +1,46 @@
+name: benchmarking
+on:
+  push:
+    branches:
+      - main
+      - release/**
+  pull_request:
+    paths:
+      - '.github/workflows/benchmarking.yml'
+      - '!**/*.md'
+      - '!**/class-diagram.svg'
+      - 'dart/**'
+      - 'flutter/**'
+      - 'benchmarking/**'
+
+jobs:
+  android:
+    name: Collect Android metrics
+    runs-on: macos-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: subosito/flutter-action@f2c4f6686ca8e8d6e6d0f28410eeef506ed66aff # pin@v2.18.0
+
+      - name: build sample-flutter
+        working-directory: ./benchmarking/sample_flutter
```

Comment:
Can we avoid adding another full-fledged sample app to the repo and instead use `flutter create`? That also saves us from updating every now & then.

Reply:
good point, thank you

Reply:
you can pass `--no-codesign` (or some such) which would make it easier. Although it's fair to check first if it affects the size but that can be done locally on any app

Reply:
@vaind I just saw the [Metrics PR](https://github.com/getsentry/sentry-dart/pull/1034) that you have created, which makes this here pretty much redundant, should have looked before. 🤦